Gemfile

```diff
@@ -8,6 +8,8 @@ gem "minitest"
 # We need a newish Rake since Active Job sets its test tasks' descriptions.
 gem "rake", ">= 13"
 
+gem "releaser", path: "tools/releaser"
```

Comment:
Heads up, this broke the devcontainer Docker image build, I put [this PR](https://github.com/rails/rails/pull/53053) up to fix it.